include/HubFramework/HUBViewModelBuilder.h

```diff
@@ -68,11 +69,19 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  *  The title that the view should have in the navigation bar
  *
- *  The value of this property will be assigned to the view controller's title property
+ *  This property is an alias for `navigationItem.title`.
  */
 @property (nonatomic, copy, nullable) NSString *navigationBarTitle;
```

Comment:
Do we intend for this property to live on? Or do we want to encourage that API users transition to using the `navigationItem` property?

If the latter, how about also deprecating this property like
```objc
DEPRECATED_MSG_ATTRIBUTE("Use the navigationItem property instead")
@property (nonatomic, copy, nullable) NSString *navigationBarTitle;
```